EmpleoDotNet/Views/JobOpportunity/Detail.cshtml

```diff
@@ -8,15 +8,14 @@
     ViewBag.Description = Regex.Replace(Model.Description, @"<(.|\n)*?>", "");
     ViewBag.ImageUrl = Model.CompanyLogoUrl;
 }
-
-@section styles{
-    <style>
-        .job-title h5 {
-            color: #14b1bb;
-            margin-top: 8px;
-            margin-bottom: 8px;
-        }
-    </style>
+@section scripts
+{
+    <script type="text/javascript">
+        var $urls = {
+            LikeJobOpportunity: '@Url.Action("Like")'
+        };
+    </script>
+    @Scripts.Render("~/Scripts/JobOpportunity/Likes.js")
```

Comment:
Si necesitas tener alguna variable server side disponible por javascript, utiliza mejor un input hidden. Esto evita que tengas que utilizar variables globales para accesar a la información.

[Aquí] hay una descripción de porque utilizar variables globales no es una buena idea.(http://stackoverflow.com/questions/2613310/ive-heard-global-variables-are-bad-what-alternative-solution-should-i-use#answer-2613647)